pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
Why do we have 'maxderivs' hardcoded in here? It should be easy to check how many of a certain prefix parameter we have in the model_builder class, and then pass that number on. Right now, this is an unnecessary limitation IMO.

Is there a limitation such that we don't do this right now?